src/components/AddBook.js

```diff
@@ -1,23 +1,29 @@
-import store from '../redux/configureStore';
+import { useDispatch } from 'react-redux';
 import { addBookObject } from '../redux/books/books';
 
-function addABook() {
-  const title = document.getElementById('title');
-  const author = document.getElementById('author');
-  if (title.value !== '' && author.value !== '') {
-    store.dispatch(addBookObject(title.value, author.value));
-  }
-}
-
 function AddBook() {
+  const dispatch = useDispatch();
   return (
     <>
       <form className="addBook">
         ADD NEW BOOK
         <br />
         <input type="text" id="title" placeholder="Book Title" name="title" />
         <input type="text" id="author" placeholder="Author" name="author" />
-        <button type="button" onClick={addABook}>Add Book</button>
+        <button
+          type="button"
+          onClick={() => {
+            const title = document.getElementById('title');
+            const author = document.getElementById('author');
```

Comment:
- [ ] Great work here, I know there are many ways to do this, but the project requirement says you should use the local state to keep track of changes to the values of the input fields available. Kindly use `useState` and an eventlistener `onChange` to achieve this. You can look at this [pluralsight](https://www.pluralsight.com/guides/handling-multiple-inputs-with-single-onchange-handler-react) tutorial to have an idea of how to achieve this. 👍